.github/ISSUE_TEMPLATE/schema-change.md

```diff
@@ -0,0 +1,75 @@
+- [ ] Create a PR which introduces the schema changes and includes the following
+  - [ ] Changes to [relevant types](https://github.com/mozilla/nimbus-shared/tree/main/types)
+  - [ ] Changes to their [associated tests](https://github.com/mozilla/nimbus-shared/tree/main/test)
+  - [ ] New [sample data](https://github.com/mozilla/nimbus-shared/tree/main/data) demonstrating
+        their use for each of desktop/mobile
+- [ ] Send an email to the [Services RFCs Mailing List](mailto:services-rfcs@mozilla.com) with a
+      link to the PR requesting feedback
+- [ ] Receive feedback, update the PR as necessary, and commit it
+- [ ] Create a [release](https://mozilla.github.io/nimbus-shared/dev/deployments) including the new
+      changes
+- [ ] File tickets/epics for all Nimbus clients to implement the changes
+  - [ ] [Nimbus Desktop](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Nimbus%20Desktop%20Client)
+  - [ ] [Nimbus Mobile](https://mozilla-hub.atlassian.net/jira/software/c/projects/SDK/boards/154/backlog)
+  - [ ] Add all filed tickets/epics in a comment below
+- [ ] For deprecated fields, file a ticket in
+      [Nimbus Engineering](https://mozilla-hub.atlassian.net/jira/software/c/projects/EXP/boards/222/backlog)
+      to add a warning to the Nimbus UI that the field will be deprecated and provide guidance about
+      how experiment/feature owners should migrate away from their dependnece on the deprecated
+      field
+
+### When Client Changes Are Complete
+
+- [ ] File tickets/epics for all Nimbus Clients for the new changes to be QAed by supplying the new
+      sample data to the relevant (desktop/mobile) client(s) and verifying the intended behaviour
+  - [ ] [Nimbus Desktop](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Nimbus%20Desktop%20Client)
+  - [ ] [Nimbus Mobile](https://mozilla-hub.atlassian.net/jira/software/c/projects/SDK/boards/154/backlog)
+  - [ ] Add all filed tickets/epics in a comment below
+
+### When Client QA is Complete
+
+- [ ] Add a comment below with a QA report that includes
+  - [ ] Which version of each client application was tested
+  - [ ] Link to any bugs which were detected in testing
+
+### When Bugs Are Resolved and Clients Are Released
+
+- [ ] Comment below indicating which version of each client application includes the changes
+- [ ] File tickets/epics in
+      [Nimbus Engineering](https://mozilla-hub.atlassian.net/jira/software/c/projects/EXP/boards/222/backlog)
+      to implement the changes in Experimenter and add them in a comment below, including:
+  - [ ] Changes to database models
+  - [ ] Changes to relevant APIs
+  - [ ] Changes to UI
+  - [ ] For deprecated fields, make sure no currently active experiments still depend on them before
+        removing them
+- [ ] File tickets/epics in [Jetstream](https://github.com/mozilla/jetstream/issues/new) to adapt it
+      to any incoming changes that affect its analysis
```

Comment:
If a new release means changes are getting deployed, then it might make sense to file a Jetstream ticket before deploying breaking changes and (maybe block the deploy until changes have been made to jetstream?). Just so that we don't run into unnecessary analysis issues.

Reply:
Yeah I was trying to think about how to manage the order dependency there, but I think it's janky in either direction.  

Either for a new field, Jetstream updates first, Experimenter isn't updated, so Jetstream looks for a non existing field and breaks.  Or for removing a field, Experimenter updates first, Jetstream depends on it and breaks.  So I think it's just a matter of coordinating them as close together as possible without strictly enforcing an order dependency?  

Unless we explicitly state an order dependency depending on whether you're adding or removing a field but I didn't want to get that deep into the weeds on a first pass.

The other thing is this is a living process so when we start implementing it, we can see how it goes in real life and update as necessary.  Does that seem reasonable?

Reply:
Yeah, sounds good!